.zenodo.json

```diff
@@ -0,0 +1,40 @@
+{
+  "creators": [
+    {
+      "affiliation": "Dartmouth College, Hanover, NH, United States",
+      "name": "Halchenko, Yaroslav O.",
+      "orcid": "0000-0003-3456-2493"
+    },
+    {
+      "affiliation": "University of Massachusetts Medical School, Worcester, MA, United States",
+      "name": "Haselgrove, Christian",
+      "orcid": "0000-0002-4438-0637"
+    },
+    {
+      "affiliation": "TCG, Inc, Washington, DC, United States",
+      "name": "Travers, Matt", 
+      "orcid": "0000-0001-5456-5371"
+    }
+  ],
+  "grants": [
+    {
+      "id": "1P41EB019936", 
+       "links": {
+        "self": "https://projectreporter.nih.gov/project_info_description.cfm?aid=8999833"
+       }, 
+       "title": "Center for Reproducible Neuroimaging Computation (CRNC)", 
+       "funder": {
+         "name": "NIH National Institute of Biomedical Imaging and Bioengineering (NIBIB)"
+       }
+    }
+  ],
```

Comment:
is that a freeform or you found it somewhere on https://developers.zenodo.org/#representation where I do not see?

any entry zenodo cannot process would trip it and not generate a record

Reply:
I guess the answer was "it was freeform" since a followup commit has removed it (although probably could have tried to fix it to be proper according to zenodo... ok, later)